Vidar/src/main/cpp/RobotContainer.cpp

```diff
@@ -86,8 +90,8 @@ void RobotContainer::ConfigureButtonBindings()
     frc2::Button coBumperLeft {[this]{return coDriver.GetRawButton(5);}};
     frc2::Button coBumperRight {[this]{return coDriver.GetRawButton(6);}};
     coA.ToggleWhenPressed(FlywheelShoot(mShooter, 0.75));
-    coB.ToggleWhenPressed(FlywheelShoot(mShooter, 0.75));
-    coY.ToggleWhenPressed(FeederShoot(mFeeder, mShooter, 0.5, units::second_t(4.0)));
+    coB.ToggleWhenPressed(FlywheelShoot(mShooter, 0.25));
```

Comment:
Using symbols in place of names will help to understand the meaning of the numbers.

constexpr double kShooterSpeed{0.25};
 coB.ToggleWhenPressed(FlywheelShoot(mShooter, kShooterSpeed));